packages/renderer/src/lib/container/ContainerColumnActions.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+import { ErrorMessage } from '@podman-desktop/ui-svelte';
+
+import ComposeActions from '../compose/ComposeActions.svelte';
+import PodActions from '../pod/PodActions.svelte';
+import ContainerActions from './ContainerActions.svelte';
+import type { ContainerGroupInfoUI, ContainerInfoUI } from './ContainerInfoUI';
+import { ContainerGroupInfoTypeUI } from './ContainerInfoUI';
+
+export let object: ContainerInfoUI | ContainerGroupInfoUI;
+</script>
+
+{#if object.type === ContainerGroupInfoTypeUI.POD && object.engineId && object.id && object.shortId && object.status && object.engineName && object.humanCreationDate && object.created}
```

Comment:
I can see that we often repeat the check `type === ContainerGroupInfoTypeUI.POD`, might be interesting to have an utility method

isPod(object: unknown): object is ContainerGroupInfoUI or something similar ? (not in this PR, just a general feeling)

Reply:
Yeah, I think we do this (and compose) several places in the code so might be nice to do this.